.travis.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       python: 3.6
     - stage: verify
       env: PURPOSE='Automation'
-      script: ./scripts/ci/test_automation.sh
+      script: travis_wait 60 ./scripts/ci/test_automation.sh
```

Comment:
Add to both 3.6 and 2.7? Is 60 too large?

Reply:
Oooooh, haha, I guess that's why it's failing on Python 2.  Yes, 60 is too large. I'll go with 30.